### ISSUE TITLE
Extend search results list view to show full-text search results

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/spotlight#1175]
Number of hits per document and context of each hit (perhaps up to some limit).

**Comment by cbeer** [comment]
We should check if solr even provides the number of hits per document. It's very possible it won't.

**Comment by ggeisler** [comment]
To keep the search results view compact, I propose we initially hide the full-text results content and show it only when the user chooses to expand it (http://getbootstrap.com/javascript/#collapse). 

If the user searches all fields, or selects the "Document text" field explicitly from the fielded search, we show a link below each result item's metadata fields (if that item has matches in the document text) that toggles the full-text match content. 

Note 1 in the mockup below is the suggested text for the toggle link. If we can get the number of hits per document from Solr, then I suggest we use the text in Note 2 of the mockup. For the toggle icons, I suggest we use the same `glyphicon-chevron-right` and `glyphicon-chevron-down` icons used for the facet toggles.

![salt-wireframes-2015-10-09_pdf__page_7_of_9_](https://cloud.githubusercontent.com/assets/101482/10401927/fb7049ec-6e76-11e5-965f-dc8d3d9cb2be.png)

When the user clicks the toggle link, we expand the full-text preview content so it looks like the mockup below:

- Show up to the first five snippets for the document
- Highlight the search term, and use a window of 120 characters on either side of the term (that's a first guess, we can adjust the window after seeing how it looks)

![salt-wireframes-2015-10-09_pdf__page_8_of_9_](https://cloud.githubusercontent.com/assets/101482/10402005/81c8f516-6e77-11e5-80f7-d7b2a51843b1.png)

**Comment by ndushay** [comment]
is this done?

**Comment by cbeer** [comment]
:+1: I think we were waiting on @snydman to decide how much further we should try to push on the hit-highlighting, but let's make that a new ticket.
